distributed/protocol/compression.py

```diff
@@ -143,7 +158,8 @@ def maybe_compress(payload, min_size=1e4, sample_size=1e4, nsamples=5):
     else:
         nbytes = len(payload)
 
-    if blosc and type(payload) is memoryview:
+    if blosc and type(payload) is memoryview and payload.itemsize > 1:
+        # Blosc does itemsize-aware shuffling, resulting in better compression
```

Comment:
It also supports memoryviews in older lz4 versions (on which versions is this active?  I get errors even fairly recently) and uses multithreading.  I'm inclined to use blosc for memoryviews if available.  Is there any reason not to use it?